app/channels/users_channel.rb

```diff
@@ -0,0 +1,12 @@
+class UsersChannel < ApplicationCable::Channel
+  def subscribed
+    room = Room.find(params[:room_id])
+    stream_for room
+  end
+
+  def unsubscribed
+    room_id = current_user.room_id
```

Comment:
does `current_user` come in scope here because somehow `ApplicationCable::Connection` is in scope? I don't fully grok how that relationship works.

Reply:
It does; I'm not 100% sure how that ends up working.  We'll have to dig around the action cable code a bit sometime.